Dockerfile.kubeadm.iso

```
@@ -92,9 +92,9 @@ RUN ARCH=$(uname -m); \
 RUN ARCH=$(uname -m); \
     if [[ $ARCH == "aarch64" ]]; then ARCH="arm64"; fi; \
     zypper --non-interactive install -- \
-      kubernetes-kubeadm \
-      kubernetes-kubelet \
-      kubernetes-client \
+      kubernetes1.28-kubeadm \
+      kubernetes1.28-kubelet  \
```

minor minor (extra space slipped in)
```suggestion
      kubernetes1.28-kubelet \
```